group_vars/all.yml

```diff
@@ -0,0 +1,57 @@
+---
+# vars file for eessi project
+
+#cvmfs_geo_license_key: 
+
+# Defaults for eessi-hpc.org config repo, syntax for each key is the same as that of cvmfs_<key | pluralize>
+galaxy_cvmfs_config_repo:
```

Comment:
Can we avoid using `galaxy`?

Reply:
Same here: renamed this to eessi_cvmfs_config_repo, but to make this work I had to add "galaxy_cvmfs_config_repo = eessi_cvmfs_config_repo" lines to the bottom of this file.